pyproject.toml

```diff
@@ -40,8 +40,6 @@ Homepage = "https://github.com/pymodbus-dev/pymodbus/"
 Discord = "https://discord.gg/vcP8qAz2"
 
 [project.scripts]
-"pymodbus.console" = "pymodbus.repl.client.main:main"
```

Comment:
Thinking about your explanation, would it not be possible to leave this in, by having 2 small mains: repl_client.py and repl_server.py that loads the repl library conditionally (like we do with pyserial) and calls the repl package ?

Reply:
It is part of the `repl` repo, the command line options will be available automatically if we install the `repl` option.

Reply:
AAAh I did not know that, super !!!!